separate-memory-transfer/README.md

```diff
@@ -4,10 +4,11 @@ An example in which we spawn a worker and transfer a buffer back and forth betwe
 and the worker.
 
 To run this example, `cd` into this directory and then:
-```rust
-> cargo build --release
-> wasm-bindgen ../target/wasm32-unknown-unknown/release/separate_memory_transfer.wasm --target=no-modules --out-dir=pkg
-> python ../server.py
+```sh
+cargo install --version 0.2.81 wasm-bindgen-cli --force
```

Comment:
```suggestion
cargo install --version 0.2.81 wasm-bindgen-cli --force # if you aren't getting it from the nix flake
```

(For people using nix, the `flake.nix`/`flake.lock` files already provide the correct version of `wasm-bindgen-cli`)